app/cdap/components/AddTransformation/ColumnTable/index.tsx

```diff
@@ -0,0 +1,105 @@
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import React from 'react';
+import { useStyles } from 'components/AddTransformation/styles';
+import T from 'i18n-react';
+import TableRowWidget from 'components/AddTransformation/ColumnTable/Components/TableRow';
+import { IColumnTableProps } from 'components/AddTransformation/ColumnTable/types';
+
+export default function({
+  columns,
+  directiveFunctionSupportedDataType,
+  onSingleSelection,
+  selectedColumns,
+  dataQualityValue,
+  isSingleSelection,
+  handleDisableCheckbox,
+  onMultipleSelection,
+}: IColumnTableProps) {
+  const classes = useStyles();
+
+  return (
+    <TableContainer component={Box}>
+      <Table aria-label="recipe steps table" className={classes.tabledisplayStyles}>
+        <TableHead>
+          <TableRow className={`${classes.recipeStepsTableRowStyles} ${classes.rowsOfTable}`}>
+            <TableCell
+              classes={{
+                head: `${classes.recipeStepsTableHeadStyles}`,
+              }}
+            ></TableCell>
+            <TableCell
+              classes={{
+                head: `${classes.recipeStepsTableHeadStyles}`,
+              }}
+            >
+              {T.translate('features.WranglerNewUI.GridPage.addTransformationPanel.columns')}
+            </TableCell>
+            <TableCell
+              classes={{
+                head: `${classes.recipeStepsTableHeadStyles}`,
+              }}
+            >
+              {T.translate('features.WranglerNewUI.GridPage.addTransformationPanel.nullValues')}
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {columns?.length > 0 &&
+            columns.map((eachColumn, index) =>
+              directiveFunctionSupportedDataType?.includes('all') ? (
+                <TableRowWidget
+                  onSingleSelection={onSingleSelection}
+                  selectedColumns={selectedColumns}
+                  dataQualityValue={dataQualityValue}
+                  isSingleSelection={isSingleSelection}
+                  handleDisableCheckbox={handleDisableCheckbox}
+                  onMultipleSelection={onMultipleSelection}
+                  index={index}
+                  eachColumn={eachColumn}
+                />
+              ) : directiveFunctionSupportedDataType?.includes(
+                  eachColumn?.type[0]?.toLowerCase()
+                ) ? (
+                <TableRowWidget
+                  onSingleSelection={onSingleSelection}
+                  selectedColumns={selectedColumns}
+                  dataQualityValue={dataQualityValue}
+                  isSingleSelection={isSingleSelection}
+                  handleDisableCheckbox={handleDisableCheckbox}
+                  onMultipleSelection={onMultipleSelection}
+                  index={index}
+                  eachColumn={eachColumn}
+                />
+              ) : (
+                <></>
```

Comment:
if we don't have any components in else, remove it and update it with && - remove empty fragment